src/main/java/net/zhuoweizhang/raspberryjuice/RemoteSession.java

```diff
@@ -469,6 +610,47 @@ protected void handleCommand(String c, String[] args) {
 					send("Fail");
 				}
 				
+			// entity.getEntities
+			} else if (c.equals("entity.getEntities")) {
+				int distance = 10;
```

Comment:
Why 10 as a default? Should the default be made within the calling [python] library and make it mandatory in the message?

Reply:
I agree. It's better to make distance parameter as mandatory.